app/(modelsMap)/models/modeling.tsx

```diff
@@ -12,6 +14,31 @@ import { useLatestItemsByCollectionIdsQuery, useRootCatalogQuery } from "./stac-
 import { EdrTable } from "./table"
 import { Layer, LoadedData } from "./types"
 
+export const ModelingPage = () => {
+  const [loading, setIsLoading] = useState(true)
+
+  useEffect(() => {
+    setIsLoading(false)
+  }, [])
+
+  return (
+    <>
+      <Row>
+        <Col md={3} style={{ padding: 0 }}>
+          <StacCatalogRoot />
+        </Col>
+        <Col md={9} style={{ paddingLeft: 0 }}>
+          {!loading && <StacMap />}
```

Comment:
Is this pattern to delay drawing the map until it's on the client?

Reply:
Yes, thats correct

Reply:
Ah, missed your comment on the issue. Feel free to merge when you are ready.